sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/models/Status.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.digitaltwins.core.implementation.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/** Defines values for Status. */
+public enum Status {
+    /** Enum value notstarted. */
+    NOTSTARTED("notstarted"),
+
+    /** Enum value running. */
+    RUNNING("running"),
+
+    /** Enum value failed. */
+    FAILED("failed"),
+
+    /** Enum value succeeded. */
+    SUCCEEDED("succeeded"),
+
+    /** Enum value cancelling. */
+    CANCELLING("cancelling"),
+
+    /** Enum value cancelled. */
+    CANCELLED("cancelled");
```

Comment:
This is admittedly more of a comment related to the backend implementation, but per Azure REST API guidelines, this terminal state should be spelled `Canceled`, not `Cancelled` - i.e., the common US spelling rather than UK. Same applies to Cancelling vs Canceling. I don't think that has any implications, though, since the generated code does not seem to tap into any common, shared async polling logic.

Reference: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs

Reply:
Noted.